### PR TITLE
GLDS: Get metrics of `className` usage in props

### LINKED ISSUE #71068: Usage of className in props
```[tasklist]
### Tasks
- [ ] Get metrics
- [ ] Add them to a dashboar
```

### PR DESCRIPTION
**Pull request by eledobleefe**
**What is this feature?**

We need this to track the usage of `className` in props in a dashboard.

**Why do we need this feature?**

To get metrics of `className` in props and be aware if we accomplish the goal of reducing the usage of classes.

**Who is this feature for?**

Saga Design System mainly

**Which issue(s) does this PR fix?**:

Fixes #71068 

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: scripts/ci-frontend-metrics.sh [hunk 1]
```diff
@@ -6,6 +6,7 @@ ACCESSIBILITY_ERRORS="$(grep -oP '\"errors\":(\d+),' pa11y-ci-results.json | gre
 DIRECTIVES="$(grep -r -o  directive public/app/ | wc -l)"
 CONTROLLERS="$(grep -r -oP 'class .*Ctrl' public/app/ | wc -l)"
 LEGACY_FORMS="$(grep -r -oP 'LegacyForms;' public/app | wc -l)"
+CLASSNAME_PROP="$(grep -r -o -E --include="*.ts*" "\.*.className\W:.*;" public/app | wc -l)"
```
Comment (academo) [inline scripts/ci-frontend-metrics.sh:9]: do you want to include test files here?

Comment (eledobleefe) [reply scripts/ci-frontend-metrics.sh:9]: Currently, we just want to look for `className` in props in `public/app`.

**Comment by eledobleefe** [comment]
> Wouldn't this have quite a significant overlap with emotion CSS imports?

They can be similar but we can test them now and if we are not happy with the results or the script does not search for what we are really expecting, we can rethink them.

File: scripts/ci-frontend-metrics.sh (hunk 1 above)
Comment (academo) [reply scripts/ci-frontend-metrics.sh:9]: @eledobleefe and test files? is that something that's relevant for your stats too?

Comment (tskarhed) [reply scripts/ci-frontend-metrics.sh:9]: If there are any classNames in test files, I think that information would be highly relevant, as it is not expected.

Comment (eledobleefe) [reply scripts/ci-frontend-metrics.sh:9]: @tskarhed I agree knowing if there are any `classNames` in test files could be interesting but we are just counting the number of files using `className` as a prop, so we don't have a way of distinguishing if that file is a test or not.

**Review by tskarhed — APPROVED** [review]

**Review by JoaoSilvaGrafana — APPROVED** [review]

**Pull request merged**